src/Redux/api.tsx

```diff
@@ -466,6 +466,8 @@ const routes = {
   },
   getPatient: {
     path: "/api/v1/patient/{id}/",
+    TRes: Type<any>(),
+    TBody: Type<{ id: string }>(),
```

Comment:
Does this API require a body to be passed in the request?

Reply:
No, I removed it.